endpoints/web.py

```diff
@@ -897,3 +898,60 @@ def redirect_to_namespace(namespace):
         return redirect(url_for("web.org_view", path=namespace))
     else:
         return redirect(url_for("web.user_view", path=namespace))
+
+
+@web.route("/api/v1/user/initialize", methods=["POST"])
+def userInitialize():
+    """
+    Create initial user in an empty database
+    """
+
+    # Ensure that we are using database auth.
+    if app.config["AUTHENTICATION_TYPE"] != "Database":
+        response = jsonify({"message": "Cannot initialize user in a non-database auth system"})
+        response.status_code = 400
+        return response
+
+    # if model.user.get_active_user_count() != 0:
```

Comment:
is this intentionally commented out? can we use the same function as in the `user.py` endpoint defined above?

Reply:
intentional yes during my testing so i don't have to reset db constantly :)